openedx/core/djangolib/model_mixins.py

```diff
@@ -0,0 +1,18 @@
+"""
+Custom Django Model mixins.
+"""
+from django.db import models
+
+
+class DeprecatedModel(models.Model):  # pylint: disable=model-missing-unicode
```

Comment:
This class (and the models that use it in `verify_student/models.py`) is intended to help with the common scenario of deleting a model.

The models are now unusable in code, but the tables don't need to be migrated. I've added a note to the [Hawthorn](https://openedx.atlassian.net/wiki/display/COMM/Hawthorn) wiki page to address this in the next openedx release.